internal/query/message_text.go

```diff
@@ -161,12 +161,20 @@ func (q *Queries) DefaultMessageTextByTypeAndLanguageFromFileSystem(messageType,
 func (q *Queries) CustomMessageTextByTypeAndLanguage(ctx context.Context, aggregateID, messageType, language string) (*MessageText, error) {
 	stmt, scan := prepareMessageTextQuery()
 	query, args, err := stmt.Where(
-		sq.Eq{
-			MessageTextColAggregateID.identifier(): aggregateID,
-			MessageTextColType.identifier():        messageType,
-			MessageTextColLanguage.identifier():    language,
+		sq.Or{
+			sq.Eq{
+				MessageTextColAggregateID.identifier(): aggregateID,
+				MessageTextColType.identifier():        messageType,
+				MessageTextColLanguage.identifier():    language,
+			},
+			sq.Eq{
+				MessageTextColAggregateID.identifier(): domain.IAMID,
+				MessageTextColType.identifier():        messageType,
+				MessageTextColLanguage.identifier():    language,
+			},
```

Comment:
this query doesn't do what it should
```sql
WHERE 
    (
        zitadel.projections.message_texts.aggregate_id = $1 
        AND zitadel.projections.message_texts.language = $2 
        AND zitadel.projections.message_texts.type = $3 
        OR zitadel.projections.message_texts.aggregate_id = $4 
        AND zitadel.projections.message_texts.language = $5 
        AND zitadel.projections.message_texts.type = $6
    ) 
```

I think you wanted this: 
```sql
WHERE
    (
        zitadel.projections.message_texts.aggregate_id = $1 
        AND zitadel.projections.message_texts.language = $2 
        AND zitadel.projections.message_texts.type = $3 
    ) OR (
            zitadel.projections.message_texts.aggregate_id = $4 
        AND zitadel.projections.message_texts.language = $5 
        AND zitadel.projections.message_texts.type = $6
    ) 
```

Reply:
as the language and type must not change i would try to create the following stmt:
```sql
WHERE
    AND zitadel.projections.message_texts.language = $1
    AND zitadel.projections.message_texts.type = $2
    AND zitadel.projections.aggregate_id IN $3
```